docs/adr/adr-0021-from-rem-to-em.md

```diff
@@ -0,0 +1,111 @@
+
+For instance, `16px` equals to `1.6rem`. 
+
+The issue is that our components were built assuming that `16px` equals to `1rem` because it's the standard.
+This means our components looked a lot smaller on these two websites.
+
+One of the key principles of our Web Components is that they can be used in almost any context.
+Using `rem` units clearly fails this principle because they rely on a root font size they cannot control.
+
+## The solution
+
+We decided to change `rem` units to `em`.
+
+This way, our components depend on the font size inherited from their parent instead of the root.
+
+We know this solution is a little less easy to use for us since we have to make calculations everytime part of a components sets a different font size.
+Fortunately, these cases are fairly rare.
+
+The switch between the two units was done manually, and we chose to round the values as much as possible.
+This means the switch was not pixel perfect but in most cases, the pixels difference between the values computed from `rem` and `em` were insignificant.
+
+## The limits
+
+However, this solution is not perfect.
+It kind of clashes with some of the [design tokens we plan to introduce later on](https://github.com/CleverCloud/clever-components/issues/398).
+
+For some of them, like `padding`, `margin`, `line-height`, it could make sense that the size would depend on the font size.
+The bigger the font size, the more space you would need, and the opposite would also be true.
+
+For others, like `font-size`, `border-radius`, `box-shadow`, we wanted to introduce tokens with values that would be the same across all our components.
+
+For instance:
+```css
+--cc-font-size-xs: 0.9em;
+```
+
+The `em` unit defeats the goal of this kind of design tokens since its value will not be the same across all our components, depending on whether another `font-size` has been set on the element or its parent.
+
+## The wish list
+
+The ideal solution would be a relative unit which computed value would remain the same between all of our Web Components. The base value could be overridden from our theme for instance.
+
+Reading through this [cancelled proposal for a 'hem' CSS unit relative to the host element font size](https://github.com/w3c/csswg-drafts/issues/7613) and the [proposal for CSS Custom Units](https://github.com/w3c/csswg-drafts/issues/7379), it seems like custom units could be a better fit for our context than `em` but this is clearly long term planning as it is still being discussed.
+
+We will continue to follow the progress on this subject.
+
+## Resources:
+
+* [The Surprising Truth about Pixels and Accessibility by Josh W Comeau](https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#the-625-trick),
+* About using relative units, in French, <a href="https://www.lalutineduweb.fr/accessibilite-personne-ne-le-fait/" lang="fr">Accessibilité : personne ne le fait donc on s’en fiche</a>.
```

Comment:
Can we add two resources that I really like about ems:

* https://nicolas-hoizey.com/talks/2013/10/10/un-petit-pas-pour-l-em-un-grand-pas-pour-le-web/
* https://nicolas-hoizey.com/articles/2016/03/02/people-don-t-change-the-default-16px-font-size-in-their-browser/